content/management/references/license-properties/index.mdx

```diff
@@ -17,32 +17,30 @@ If you need to brush up on properties, [read more](/management/configuration-api
 
 # Available properties
 
-| Namespace       | Properties                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
-| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| **lc2**         |  [`welcome_text`](#welcome_text)                                                                                                                                                                                      |
+| Namespace   | Properties                                |
+|-------------|-------------------------------------------|
+| **privacy** | [`open_chat_noindex`](#open_chat_noindex) |
 
 <Section>
 <Text>
 
-### welcome_text
+### open_chat_noindex
 
-Contains the welcome message text. Appears only if `"welcome_message": true`.
+Indicates if the server should return chat page (direct chat link and I-Frame inside widget) with response header `X-Robots-Tag: noindex` to prevent crawlers indexing the page.
```

Comment:
```suggestion
Indicates if the server should return the chat page (a direct chat link and an Iframe inside the chat widget) with the `X-Robots-Tag: noindex` response header to prevent crawlers from indexing the page.
```